.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    target_branch: "master"
```

Comment:
In [the documentation](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates#target-branch), it is called `target-branch`.
```suggestion
    target-branch: "master"
```

Reply:
:man_facepalming: Thanks!